pkg/segment/structs/evaluationstructs.go

```diff
@@ -1132,6 +1132,49 @@ func handleSplit(self *MultiValueExpr, fieldToValue map[string]sutils.CValueEncl
 
 	return stringsList, nil
 }
+func handleMVFilter(self *MultiValueExpr, fieldToValue map[string]sutils.CValueEnclosure) ([]string, error) {
+	if self.MultiValueExprParams == nil || len(self.MultiValueExprParams) != 1 || self.MultiValueExprParams[0] == nil {
+		return []string{}, fmt.Errorf("MultiValueExpr.Evaluate: mvfilter requires one multiValueExpr argument")
+	}
+	if self.Condition == nil {
+		return []string{}, fmt.Errorf("MultiValueExpr.Evaluate: mvfilter requires a condition")
```

Comment:
nit:
```suggestion
		return []string{}, fmt.Errorf("handleMVFilter: mvfilter requires a condition")
```

Same comment elsewhere